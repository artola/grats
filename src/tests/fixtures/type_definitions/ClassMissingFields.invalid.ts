/** @gqlType */
export default class SomeType {}
