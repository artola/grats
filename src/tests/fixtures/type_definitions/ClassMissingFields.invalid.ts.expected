-----------------
INPUT
----------------- 
/** @gqlType */
export default class SomeType {}

-----------------
OUTPUT
-----------------
src/tests/fixtures/type_definitions/ClassMissingFields.invalid.ts:2:22 - error: Type `SomeType` must define one or more fields.

Add a field by adding `/** @gqlField */` above a field, property, attribute or method of this type, or above a function that has `SomeType` as its first argument.

2 export default class SomeType {}
                       ~~~~~~~~
